Fix integration job when run on schedule

The integration job errors out when it is run on a schedule.

Logs: https://productionresultssa4.blob.core.windows.net/actions-results/db66a1cb-1ebc-42d3-8671-a9962a0a4382/workflow-job-run-24c4d16e-ab09-5d3c-a0d3-0112ee19eacb/logs/job/job-logs.txt?rsct=text%2Fplain&se=2025-09-16T09%3A03%3A49Z&sig=rd2%2Bmcv5tfaRwOGm294WoEGBL%2BvO5wpzY9GVsJLEYAo%3D&ske=2025-09-16T17%3A07%3A58Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2025-09-16T05%3A07%3A58Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2025-05-05&sp=r&spr=https&sr=b&st=2025-09-16T08%3A53%3A44Z&sv=2025-05-05



Error: 

Run COMMENT_BODY=$(cat consolidated_report.md)
GraphQL: Could not resolve to a PullRequest with the number of 9745. (repository.pullRequest)
Error: Process completed with exit code 1.

yml file: .github/workflows/integration-runner.yml

--- a/.github/workflows/integration-runner.yml
+++ b/.github/workflows/integration-runner.yml
@@ -163,6 +163,7 @@ jobs:
           path: consolidated_report.md
 
       - name: Create consolidated PR comment
+        if: github.event_name == 'pull_request'
         run: |
           COMMENT_BODY=$(cat consolidated_report.md)
           # Use GitHub CLI to create comment

